litellm/llms/vertex_httpx.py

```diff
@@ -1050,6 +1053,8 @@ def completion(
                 data["safetySettings"] = safety_settings
             if generation_config is not None:
                 data["generationConfig"] = generation_config
+            if cached_content is not None:
+                data["cachedContent"] = cached_content
```

Comment:
can you share an example of a call with this that works? 

I thought cachedcontent was a specific client that had to be pulled?

Reply:
can i see what your code with creating the cache + using litellm looks like? 

Can't we just add it as a cache option? like s3 cache

Reply:
oh - why use the proxy here? 

since you're already using the vertex endpoint

Reply:
fyi might be relevant here - if you want to run the cache create call through proxy https://docs.litellm.ai/docs/proxy/pass_through